packages/utils/src/api-client.js

```diff
@@ -46,6 +46,7 @@ class ApiClient {
    * @return {URL}
    */
   _normalizeURL(url) {
+    if (this._rootURL) return new this._URL(this._rootURL + url, this._rootURL);
```

Comment:
you can go ahead and revert this for the tests I'll make a pass and clean this up with some tests too :)

Reply:
reverted and squashed my changes